pages/common/csvformat.md

```diff
@@ -0,0 +1,24 @@
+# csvformat
+
+> Convert a CSV file to a custom output format.
+> Included in csvkit.
+
+- Convert to a tab-delimited file:
+
+`csvformat -T {{data.csv}}`
+
+- Convert delimiters to a custom character:
+
+`csvformat -D "{{custom_character}}" {{data.csv}}`
+
+- Convert line endings to carriage return + line feed:
+
+`csvformat -M "{{\r\n}}" {{data.csv}}`
+
+- Convert to minimal use of quote characters:
+
+`csvformat -U 0 {{data.csv}}`
+
+- Convert to maximum use of quote characters:
```

Comment:
I'm not sure I get what's the difference then, and what "maximum" and "minimum" mean here.